google/cloud/aiplatform/training_jobs.py

```diff
@@ -3346,3 +3347,255 @@ def _run(
+            )
+
+            training_task_inputs_dict = AutoMlTextExtractionInputs().to_value()
+        elif self._prediction_type == "sentiment":
+            training_task_definition = (
+                schema.training_job.definition.automl_text_sentiment
+            )
+
+            if self._sentiment_max is None:
+                raise ValueError(
+                    "Tge sentiment_max parameter must be provided for a prediction_type of 'sentiment'"
+                )
+
+            training_task_inputs_dict = AutoMlTextSentimentInputs(
+                sentiment_max=self._sentiment_max
+            ).to_value()
+        else:
+            raise ValueError(
+                "Prediction type must be one of 'classification', 'extraction', or 'sentiment'."
+            )
+
+        self._training_task_definition = training_task_definition
+        self._training_task_inputs_dict = training_task_inputs_dict
+
+    def run(
+        self,
+        dataset: datasets.Dataset,
+        training_fraction_split: float = 0.8,
+        validation_fraction_split: float = 0.1,
+        test_fraction_split: float = 0.1,
+        model_display_name: Optional[str] = None,
+        sync: bool = True,
+    ) -> models.Model:
+        """Runs the training job and returns a model.
+
+        Data fraction splits:
+        Any of ``training_fraction_split``, ``validation_fraction_split`` and
+        ``test_fraction_split`` may optionally be provided, they must sum to up to 1. If
+        the provided ones sum to less than 1, the remainder is assigned to sets as
+        decided by AI Platform. If none of the fractions are set, by default roughly 80%
+        of data will be used for training, 10% for validation, and 10% for test.
+
+        Args:
+            dataset (datasets.Dataset):
+                Required. The dataset within the same Project from which data will be used to train the Model. The
+                Dataset must use schema compatible with Model being trained,
+                and what is compatible should be described in the used
+                TrainingPipeline's [training_task_definition]
+                [google.cloud.aiplatform.v1beta1.TrainingPipeline.training_task_definition].
+            training_fraction_split: float = 0.8
+                Required. The fraction of the input data that is to be
+                used to train the Model. This is ignored if Dataset is not provided.
+            validation_fraction_split: float = 0.1
+                Required. The fraction of the input data that is to be
+                used to validate the Model. This is ignored if Dataset is not provided.
+            test_fraction_split: float = 0.1
+                Required. The fraction of the input data that is to be
+                used to evaluate the Model. This is ignored if Dataset is not provided.
+            model_display_name (str):
+                Optional. If the script produces a managed AI Platform Model. The display name of
```

Comment:
I think the "If the script produces..." can be removed because this will always create a managed model.

Reply:
Done